tests/zfs-tests/tests/functional/xattr/xattr_common.kshlib

```diff
@@ -134,4 +134,24 @@ function create_expected_output  { # expected_output_file  contents_of_the_outpu
    do
       log_must eval "echo $line >> $FILE"
    done
- }
+}
+
+function check_mem_leaks {
+	log_note "Checking for leaks"
+	if $(sudo -E test -e "$KMEMLEAK_FILE"); then
```

Comment:
You'll probably need to adapt this a little bit for the ZTS environment, for example the sudo calls  shouldn't be needed.  And make sure to define `KMEMLEAK_FILE="/sys/kernel/debug/kmemleak"` and `KMEMLEAK_FILE` to something reasonable.

Reply:
Oh good point...